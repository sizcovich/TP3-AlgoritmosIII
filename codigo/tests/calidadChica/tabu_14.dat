6 15 16 944800
26 25 50 5144100
50 35 91 12014200
101 45 170 24268300
26 55 241 16946400
31 65 365 6019000
36 75 466 12134700
41 85 635 21109200
485 95 766 87781300
626 105 980 129639100
6 15 16 713000
11 25 50 2716200
50 35 91 12200400
101 45 170 23993900
26 55 241 18350800
226 65 365 17086300
290 75 466 32780600
401 85 635 56285600
46 95 766 33431300
51 105 980 48914300
