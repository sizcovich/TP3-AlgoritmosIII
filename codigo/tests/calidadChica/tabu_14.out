6 1 15 
11 1 23 
16 1 28 
21 1 36 
26 1 15 
31 1 15 
36 1 20 
41 1 12 
46 1 70 
6 1 5 
26 5 10 2 3 4 15 
50 7 11 7 10 16 25 31 34 
101 10 15 2 5 10 20 26 28 30 35 39 
26 1 48 
31 1 29 
36 1 49 
41 1 55 
485 22 12 11 20 21 22 28 29 34 45 46 55 60 63 68 69 70 76 80 84 88 89 94 
626 25 92 2 4 9 12 13 26 30 33 36 40 47 50 53 54 61 63 64 70 72 73 83 85 89 97 
6 1 11 
11 1 9 
50 7 17 2 6 7 12 26 34 
101 10 1 4 16 24 26 27 29 31 32 45 
26 1 46 
226 15 15 11 16 21 23 24 25 28 31 40 46 50 52 60 63 
290 17 1 22 2 5 9 18 19 21 28 37 38 43 52 54 64 67 68 
401 20 1 30 9 18 20 21 24 32 37 38 41 45 46 51 56 57 59 60 66 67 
46 1 14 
51 1 59 
